server/src/modules/user/user.model.ts

```diff
@@ -31,6 +32,19 @@ export class User extends Model<User> {
   @Column({ type: DataType.ENUM(...Object.values(UserType)), allowNull: false })
   userType: UserType;
 
+  @Column({
+    type: DataType.STRING(200),
+    allowNull: false,
+    field: 'passwordHash',
+    get: function () {
+      return PasswordHash.fromHashString(this.getDataValue('passwordHash'));
+    },
+    set: function (value: PasswordHash) {
+      this.setDataValue('passwordHash', value.getHash());
+    },
+  })
+  passwordHash: PasswordHash;
```

Comment:
Nice job setting up this attribute! Looks good to me, glad you were able to use `PasswordHash` as the TypeScript type and get the getter/setter to work